package.json

```diff
@@ -145,9 +145,10 @@
     "jest-dev-server": "6.1.1",
     "jsdom": "20.0.0",
     "messageformat": "2.3.0",
-    "nuxt": "2.15.8",
+    "nuxt": "2.17.2",
     "playwright-chromium": "1.24.2",
     "release-it": "15.2.0",
+    "sass": "^1.69.7",
```

Comment:
Type error for `sass`.
It is difficult and takes time to solve with upstream, so I added sass as a temporary solution.
https://github.com/nuxt/nuxt/pull/24118